src/OrderFormAcceptanceTests.Steps/Steps/PreviewOrderSummary.cs

```diff
@@ -1,344 +1,393 @@
+        {
+            SetOrderCatalogueSectionToComplete();
+            var orderItem = new OrderItem().GenerateAdditionalServiceWithFlatPricedVariableOnDemand((Order)Context["CreatedOrder"]);
+            orderItem.EstimationPeriodId = 2;
+            orderItem.TimeUnitId = 2;
+            orderItem.Create(Test.ConnectionString);
+            Context.Add("CreatedOrderItem", orderItem);
+        }
+
+        [Given(@"an additional service with a flat price variable \(On-Demand\) order type with the quantity period per month is saved to the order")]
+        public void GivenAnAdditionalServiceWithAFlatPriceVariableOn_DemandOrderTypeWithTheQuantityPeriodPerMonthIsSavedToTheOrder()
+        {
+            SetOrderCatalogueSectionToComplete();
+            var orderItem = new OrderItem().GenerateAdditionalServiceWithFlatPricedVariableOnDemand((Order)Context["CreatedOrder"]);
+            orderItem.EstimationPeriodId = 1;
+            orderItem.TimeUnitId = 1;
+            orderItem.Create(Test.ConnectionString);
+            Context.Add("CreatedOrderItem", orderItem);
+        }
+
+        [Given(@"an additional service with a flat price variable \(Patient\) order type is saved to the order")]
+        public void GivenAnAdditionalServiceWithAFlatPriceVariablePatientOrderTypeIsSavedToTheOrder()
+        {
+            SetOrderCatalogueSectionToComplete();
+            new AdditionalServices(Test, Context).GivenTheSupplierAddedToTheOrderHasAnAdditionalServiceWithAPatientFlatPrice();
+            var orderItem = new OrderItem().GenerateAdditionalServiceOrderItemWithVariablePricedPerPatient((Order)Context["CreatedOrder"]);
+            orderItem.Create(Test.ConnectionString);
+            Context.Add("CreatedOrderItem", orderItem);
+        }
+
+        [Given(@"there are one or more Order items summarised in the Order items \(recurring cost\) table")]
+        public void GivenThereAreOneOrMoreOrderItemsSummarisedInTheOrderItemsRecurringCostTable()
+        {
+            var onDemandOrderItem = new OrderItem().GenerateOrderItemWithFlatPricedVariableOnDemand((Order)Context["CreatedOrder"]);
+            onDemandOrderItem.Create(Test.ConnectionString);
+
+            var declarativeOrderItem = new OrderItem().GenerateOrderItemWithFlatPricedVariableDeclarative((Order)Context["CreatedOrder"]);
+            declarativeOrderItem.Create(Test.ConnectionString);
+
+            var patientOrderItem = new OrderItem().GenerateOrderItemWithFlatPricedVariablePerPatient((Order)Context["CreatedOrder"]);
+            patientOrderItem.Create(Test.ConnectionString);
+        }
+
+        [Then(@"the Total cost for one year is the result of the Total cost for one year calculation")]
+        public void ThenTheTotalCostForOneYearIsTheResultOfTheTotalCostForOneYearCalculation()
+        {
+            var actual = Test.Pages.OrderForm.GetTotalAnnualCost();
+
+            const string expectedTotalAnnualCost = "1,316,535.00";

```

Comment:
not sure about having this hard coded, is there anyway we can calc it based on what we know? like ThenTheItemYearCostOfEachItemIsTheResultOfTheFlatCalculationPriceQuantityTimes12RoundedUpToTwoDecimalPlaces